genie-web/src/main/java/com/netflix/genie/web/security/SecurityConfig.java

```diff
@@ -38,6 +39,7 @@
  */
 @Configuration
 @Slf4j
+@Order(357)
```

Comment:
Where is this magic number coming from?

Reply:
Good question. This class extends the default web security stuff and its only role is to disable all security which boot turns on by default in the situation where no other security is enabled. Before it didn't have an `@Order` on it so it was inheriting the super class one (100). If any other class in the classpath also tried to provide a default security configuration (internal security for example) it would conflict. So we just chose some random unlikely to be chosen number > 100 here to make sure it took effect after any default internal security configuration.